quartz/i18n/index.ts

```diff
@@ -56,6 +56,7 @@ export const TRANSLATIONS = {
   "pt-BR": pt,
   "hu-HU": hu,
   "fa-IR": fa,
+  "pl-PL": pl,
```

Comment:
make sure to import the translation

Reply:
I am not sure what that mean. I don't know TS so I just tried to do what you listed here: https://quartz.jzhao.xyz/features/i18n

Reply:
see the import lines from L1-L18 and try to follow that format